v8go.cc

```diff
@@ -26,6 +26,7 @@ const int ScriptCompilerEagerCompile = ScriptCompiler::kEagerCompile;
 
 struct m_ctx {
   Isolate* iso;
+  StartupData* startup_data;
```

Comment:
Adding the `StartupData` to our local context as a convenient location to allow us to free that data once the isolate gets `Dispose()`.